instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/Instrumenter.java

```diff
@@ -119,8 +122,12 @@ public Context start(Context parentContext, REQUEST request) {
       spanBuilder.setStartTimestamp(startTimeExtractor.extract(request));
     }
 
+    for (SpanLinkExtractor<? super REQUEST> extractor : spanLinkExtractors) {
+      spanBuilder.addLink(extractor.extract(parentContext, request));
```

Comment:
Can extractor NOT return any link? What will happen if no link is found?

Reply:
We should have tests for both cases

Reply:
> Can extractor NOT return any link? What will happen if no link is found?

It can return `SpanContext.getInvalid()` - which will be ignored by the SDK. I'll add a comment explaining that.